### PR TITLE
Update changelog.

### PR DESCRIPTION
**Pull request by kaikai1024**
### Description of the Change

Update changelog for preparing the new version.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,45 @@ All notable changes to this project will be documented in this file. And this pr
...
+
+- [Refactor] Refactor scripts of integrate test. [@kaikai1024]
+
+### Scripts
+
+- [Fix] Node path should not consider bin path. [@rainchen]
+- [Refactor] Refactor the `cita.sh` script. [@kaikai1024]
+- [Fix] Fix amend help info and latest version. [@leeyr338]
+- [Optimization] Rename `config_example` to `default_config`. [@kaikai1024]
+
```
Comment (ouwenkg) [inline CHANGELOG.md]: * recover security audit test
* upgrade protobuf

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,45 @@ All notable changes to this project will be documented in this file. And this pr
...
+- [Fix] Fix the bug that save wrong session. [@leeyr338]
+- [Fix] Should not unwrap on handle_remote_msg. [@leeyr338]
+- [Fix] Fix the operation of repeated peer key. [@jerry-yu]
+
+### System Contracts
+
+- [Fix] Fix the import path. [@ouwenkg]
+- [Optimization] Use `protocol version` instead of version. [@kaikai1024]
+
```
Comment (ouwenkg) [inline CHANGELOG.md]: ### Executor

- [Fix] Fix memory leak

File: CHANGELOG.md (hunk 1 above)
Comment (kaikai1024) [reply CHANGELOG.md]: wait a minute.

Comment (ouwenkg) [reply CHANGELOG.md]: My fault. Just ignore this item.

File: CHANGELOG.md (hunk 2 above)
Comment (kaikai1024) [reply CHANGELOG.md]: It's already on `v0.24.1`

**Pull request merged**